composer.json

```diff
@@ -34,13 +34,9 @@
         }
     },
     "scripts": {
-        "post-root-package-install": [
-            "php -r \"file_exists('.env') || copy('.env.example', '.env');\""
-        ],
-        "post-create-project-cmd": [
-            "php artisan key:generate"
-        ],
         "post-install-cmd": [
+            "php -r \"file_exists('.env') || copy('.env.example', '.env');\"",
+            "php artisan key:generate",
```

Comment:
That's a good point, it would regenerate the key on every `composer install` however we should then add that command to the setup instructions since it wouldn't be run on install.